templates/shared/_blender-contact-us-form.html

```diff
@@ -0,0 +1,77 @@
+<section class="p-strip is-deep">
+  <div class="row">
+    <div class="col-8">
+      <h1>{{h1}}</h1>
+      <p>{{intro_text}}</p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <form action="https://pages.ubuntu.com/index.php/leadCapture/save" method="post" id="mktoForm_{{formid}}">
+
+        <fieldset class="u-no-margin--bottom">
+          <label for="CommentsAboutProject">Tell us about your project and what problem(s) you’re looking to solve.</label>
+          <textarea required id="CommentsAboutProject" name="CommentsAboutProject" maxlength="255" type="text" class="mktoField mktoRequired"></textarea>
```

Comment:
Almost there, but needs a bit more finessing:
- [ ] the `CommentsAboutProject` values on the two elements should be changed to `Comments_from_lead_c`. I think this is a field setup in Marketo, so we should keep this consistent with other contact forms on u.c that have a similar field (search the codebase for that value to see what I mean).
- [ ] `textarea` elements don't need the `type` attribute
- [ ] Marketo classes are missing from the textarea field
- [ ] it reads better if the label ends with a colon, rather than a period
- [ ] the maxlength should be increased, textareas in other contact forms use 2000, so let's go with that
```suggestion
          <label for="Comments_from_lead__c" class="mktoLabel p-heading--three">Tell us about your project and what problem(s) you’re looking to solve:</label>
          <textarea required id="Comments_from_lead__c" name="Comments_from_lead__c" class="mktoField mktoRequired" maxlength="2000"></textarea>
```

Reply:
Thanks for the feedback, it is ready for another review!